ova/provision.sh

```diff
@@ -4,17 +4,32 @@ set -exf
 repo_branch=$(echo "$1" | cut -c1-3)
 repo_baseurl=$(echo "$1" | cut -c1-2)
 WAZUH_VERSION=$1
-ELK_VERSION=$2
-STATUS_PACKAGES=$3
-DIRECTORY=$4
+OPENDISTRO_VERSION=$2
+ELK_VERSION=$3
+STATUS_PACKAGES=$4
+BRANCH=$5
+DIRECTORY="/var/ossec"
 ELK_MAJOR=`echo ${ELK_VERSION}|cut -d"." -f1`
 ELK_MINOR=`echo ${ELK_VERSION}|cut -d"." -f2`
+
 config_files="/vagrant/Config_files"
+automatic_set_ram_location="/etc/"
+libraries_files="/vagrant/Libraries/"
+
+echo "${STATUS_PACKAGES}"
+. /vagrant/Libraries/provision-opendistro.sh
 
-. /vagrant/Libraries/wazuh_functions.sh
-. /vagrant/Libraries/elastic_functions.sh
 
 # Setting wazuh default root password
+
+cp ${libraries_files}/"automatic_set_ram.sh" ${automatic_set_ram_location}
+chmod +x "${automatic_set_ram_location}/automatic_set_ram.sh"
```

Comment:
This is not necessary if the `automatic_set_ram.sh` script already has execution permissions.